engines/billing/questions.md

```diff
@@ -0,0 +1,22 @@
+Questions
+====================
+
+1. Should admin be able to see subscriptions?
+
+2. Should user be able to upgrade or downgrade subscriptions?
+
+3. Should confirmations emails be allowed?
```

Comment:
Exactly! That goes beyond Braintree' security. There are also 'email notifications' that can be sent to a customer when its subscription is 'past due'. But then again, there are securities' issues.